BUILD.md

```diff
@@ -113,7 +112,7 @@ Add the ShapeWorks and dependency binaries to the path:
 ### Examples
 *OSX* example that builds dependencies separately, then generates an XCode project for ShapeWorks:  
 ```
-$ ./superbuild.sh  --dependencies_only --build-dir=../dependencies --install-dir=../dependencies
+$ ./build_dependencies.sh --build-dir=../dependencies --install-dir=../dependencies
```

Comment:
../dependencies or ./dependencies? not consistent with the usage output of build_dependencies.sh.

Reply:
I've updated it.  The default install_dir for "build_dependencies.sh" is $PWD/dependencies/install